src/components/PostPreview/PostPreview.tsx

```diff
@@ -0,0 +1,77 @@
+import type Post from '@types/Post';
+import { Avatar } from '../Avatar';
+import Icon from '../Icon';
+import {
+  PreviewWrapper,
+  PostHeaderWrapper,
+  PostInfoWrapper,
+  PostContent,
+  UserNameWrapper,
+  AvatarWrapper,
+  PostDetailInfoWrapper
+} from './PostPreview.style';
+
+interface PostPreviewProps {
+  post: Post;
+  noneProfile: boolean;
+}
+
+const PostPreview = ({ post, noneProfile = false }: PostPreviewProps) => {
+  const { image, likes, comments, title, author, createdAt } = post;
+  const totalLikes = likes.length;
+  const totalComments = comments.length;
+  const iconDescription = [
+    { name: 'favorite', size: 12, total: totalLikes },
+    { name: 'chat', size: 12, total: totalComments }
+  ];
+  const previewTitle = title.substring(0, 100);
+  // todo: 명상 시간 데이터는 아직 못받아왔으므로 추후에 제외시켜놓은 거 추가하기
+
+  const PostDetailInfo = () => {
+    return (
+      <PostDetailInfoWrapper>
+        {createdAt}
+        {iconDescription.map((iconInfo, index) => {
+          return (
+            <div key={index}>
+              <Icon
+                name={iconInfo.name}
+                size={iconInfo.size}
+                color={'greyLight'}
+              />
+              {iconInfo.total}
+            </div>
+          );
+        })}
+      </PostDetailInfoWrapper>
+    );
+  };
+  const PostHeaderWithUser = () => {
+    return (
+      <>
+        <AvatarWrapper>
+          <Avatar
+            alt={'유저 프로필'}
+            src={image}
+            size={25}
+          />
+        </AvatarWrapper>
+        <PostInfoWrapper>
+          <UserNameWrapper>{author}</UserNameWrapper>
+          <PostDetailInfo />
+        </PostInfoWrapper>
+      </>
+    );
+  };
+
+  const PostHeader = noneProfile === true ? PostHeaderWithUser : PostDetailInfo;
```

Comment:
noneProfile로 비교되는 두 대상의 네이밍의 공통점이 잘 안보여서 어떤 비교인지 모호한 것 같아요
```suggestion
  const PostHeader = noneProfile === true ? PostHeaderWithProfile : PostHeaderNoneProfile;
```
이런식은 어떨까요?

Reply:
프로필없이 포스트 프리뷰를 보여주는 경우는 어떤 경우인가요?

Reply:
userProfile 에 들어간 경우 프로필을 보여주지 않습니다!

또 `noneProfile` 을 통해 && 연산자를 사용하여 필요한 경우 Profile 을 추가하는 방식으로 변경하였습니다!